packages/fluent-editor/src/modules/ai/index.ts

```
@@ -218,68 +717,128 @@ export class AI {
-
-      const actions = [this.DONE, this.REGENERATE, this.CLOSE]
-      actions.forEach((action) => {
-        const menuItem = document.createElement('div')
-        menuItem.className = 'ql-ai-action-item'
-        menuItem.textContent = action
-        menuItem.addEventListener('click', () => this.handleAction(action))
-        this.actionMenuEl.appendChild(menuItem)
-      })
-
-      this.wrapContainerEl.appendChild(this.actionMenuEl)
+    if(this._charCount <= this.textNumber) {
```

ESLint

[error] Expected space(s) after "if".

(style/keyword-spacing)